jira_projects_create_cards.py

```diff
@@ -0,0 +1,340 @@
+        """
+        issuetype = self._add_issue_type()
+        summary = self._add_summary()
+        description = self._add_description()
+
+        return {
+        'summary': summary,
+        'description': description,
+        'issuetype': {'name': issuetype},
+        }
+
+    def _add_issue_type(self):
+        """ Add issue type
+
+            :return: issue type
+        """
+        issuetype_list = ['Bug', 'Task']
+        while self.issuetype not in issuetype_list:
+            print(
+                'Possible issue types are: [%s]' % '; '.join(
+                    map(str, issuetype_list)
+                )
+            )
+            self.issuetype = input('Choose the issue type: ')
+        return self.issuetype
+
+    def _add_description(self):
+        """ Add issue description
+
+            :return: issue description
+        """
+        while not self.description:
+            print("Description shouldn't be empty")
+            self.description = input('Add description: ')
+
+        self.description = self.description.replace('\\n', '\n')
+        return self.description
+
+    def _add_summary(self):
+        """ Add issue summary
+
+            :return: issue summary
+        """
+        while not self.summary:
+            print("summary shouldn't be empty")
+            self.summary = input('Add summary: ')
+        return self.summary
+
+@click.command()
+@click.option('--topic', default=False, multiple=True, help='GitHub topic')
+@click.option('--description', default=False, help='New jira issue description')
+@click.option('--issuetype', default=False, help='New jira issue type')
+@click.option('--summary', default=False, help='New jira issue summary')
+@click.option('--existing_issue_copy', default=False, help='Issue to duplicate name like JRA-1330')
+@click.option('--csv_project_list', default=False, help='Give the path to your project list csv')
+@click.option('--csv_jira_card', default=False, help='Give the path to your card list csv')
+def main(
+        topic,
+        description,
+        issuetype,
+        summary,
+        existing_issue_copy,
+        csv_project_list,
+        csv_jira_card):
+    """
+        Create a Jira issue on project list retrieved in GitHub
+
+        The project list is retrieved from GitHub repositories
+        search using topic filter.
+        Multiple topics can be combined like "odoo-9,need-5-digits".
+        If you don't complete directly the CLI option, it is not a problem,
+        you will be prompted to use this tool.
+
+        If your action corresponds to the script process, you will see
+        for all your issue creation:\n
+            'Issue ok for [project_name]'
+
+        Enjoy and don't hesitate to contribute or create an issue :-)
+    """
+
+    kwargs = {
+        'topic': topic,
+        'description': description,
+        'issuetype': issuetype,
+        'summary': summary,
+        'existing_issue_copy': existing_issue_copy,
```

Comment:
@yvaucher asked for `--template` instead. Any other unattended comment compared to the old PR?